CMakeLists.txt

```diff
@@ -839,6 +842,12 @@ if(EVENT__HAVE_EPOLL)
     list(APPEND SRC_CORE epoll.c)
 endif()
 
+if(EVENT__HAVE_WEPOLL)
+    list(APPEND SRC_CORE
+        epoll.c
+        wepoll.c)
+endif()
```

Comment:
missing support for dist archive, i.e. `make dist` (this command is used to create binary archives that will be attached to releases), you need to add this files into `Makefile.am` too

Reply:
Added dist check for linux 512c88ce (but even if this will be added for windows, it will not show the problem, since it will just not include wepoll backend, so maybe some test should be added here)

Reply:
Hmm, let me know if I am piecing this together right. make dist is only available when setting up the build tools with autotools. Autotools is only available on Windows via cygwin or mingw. Therefore, libevent's Windows binary archives come from compilation on mingw?

Reply:
[dist archive](https://www.gnu.org/software/automake/manual/html_node/Preparing-Distributions.html) (created with `make dist`) creates archive that will be attached to the release, it contains all sources and does not requires from you to run `autogen.sh` (and this is the only difference that I remember)

so what you need to do is add this new files into it - [here](https://github.com/libevent/libevent/blob/15917b420d47c9fee6a15331f20f0e9f60cbcd24/Makefile.am#L297)

P.S. maybe we will refuse from using it, but this is another story

Reply:
I see. Sorry, please forgive my lack of autotools knowledge :) I still have one question though. If the archived distribution is put together by autotools, then it seems one can compile an archived distribution of libevent on Windows only via mingw. If that is correct, this PR needs work to make sure wepoll backend gets compiled in from mingw platform. I explicitly disallowed it in c08f6bd to keep things simple.

Reply:
The dist package can be done on any platform (it is basically just an archive with sources)

P.S. No worries

Reply:
Ok. I added wepoll.h to dist archive as suggested